users/pauper/config.h

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Userspace additions should be submitted separately

Reply:
Accident. Thanks for the heads-up. I'll pull it out.

Reply:
Made all requested changes.